tests/python/test_deepseek_v3.py

```diff
@@ -49,8 +50,67 @@ def default_tensor_type(dtype=torch.float32, device="cpu"):
     torch.set_default_device(prev_device)
 
 
+@contextmanager
+def cuda_profiler():
```

Comment:
This isn't used in the test as is but is kept in case the user wants to `nsys profile`.